app/views/service_worker/index.js.erb

```diff
@@ -35,10 +35,20 @@
     self.clients.claim();
   });
 
+  function includesUnsupportedPath(url) {
+    return [
+      '/%F0%9F%92%B8',         // 💸 (hiring)
+      '/admin',                // Don't run on admin dashboard.
+      '/oauth/',               // Skip oauth apps
+      '/onboarding',           // Don't run on onboarding.
+      '/users/auth',           // Don't run on authentication.
+    ].some(path => url.includes(path))
+  }
```

Comment:
Just thinking about this now, should we add all the other paths from the previous version of the service workers?

https://github.com/forem/forem/blob/37f609e5c51b0639ae915a2f5800fc8fd7a40f78/app/views/service_worker/index.js.erb#L90-L129